OSRMTextInstructions/OSRMTextInstructions.swift

```diff
@@ -121,10 +121,10 @@ public class OSRMInstructionFormatter: Formatter {
     typealias InstructionsByModifier = [String: InstructionsByToken]
     
     override public func string(for obj: Any?) -> String? {
-        return string(for: obj, legIndex: nil, numberOfLegs: nil, modifyValueByKey: nil)
+        return string(for: obj, legIndex: nil, numberOfLegs: nil, roadClasses: nil, modifyValueByKey: nil)
     }
     
-    public func string(for obj: Any?, legIndex: Int?, numberOfLegs: Int?, modifyValueByKey: ((TokenType, String) -> String)?) -> String? {
+    public func string(for obj: Any?, legIndex: Int?, numberOfLegs: Int?, roadClasses: RoadClasses?, modifyValueByKey: ((TokenType, String) -> String)?) -> String? {
```

Comment:
There's no semantic difference between an empty RoadClasses (`[]`) and nil, so let's make the type of the roadClasses argument required. The simpler version of this method above would pass in `[]`.